membersonlyevent.php

```diff
@@ -198,9 +198,17 @@ function membersonlyevent_civicrm_preProcess($formName, &$form) {
  *
  * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_navigationMenu
  */
-function membersonlyevent_civicrm_navigationMenu(&$params) {
-  _membersonlyevent_add_configurations_menu($params);
-}
+function membersonlyevent_civicrm_navigationMenu(&$menu) {
+  $membersonlyeventMenu = [
+    'label' => ts('Members Only Event Extension Configurations'),
+    'name' => 'membersonlyevent_configurations',
+    'url' => 'civicrm/admin/membersonlyevent',
+    'permission' => 'administer CiviCRM,access CiviEvent',
+    'operator' => NULL,
+    'separator' => NULL,
+  ];
+
+  _membersonlyevent_civix_insert_navigation_menu($menu, 'Administer/CiviEvent', $membersonlyeventMenu);}
```

Comment:
Pleas put the closing brace on a new line (ie. the '}' at the end of this line).